app/src/full/java/com/celzero/bravedns/adapter/FirewallAppListAdapter.kt

```diff
@@ -501,4 +518,10 @@ class FirewallAppListAdapter(
     private suspend fun ioCtx(f: suspend () -> Unit) {
         withContext(Dispatchers.IO) { f() }
     }
+
+    private fun getColorFromAttr(attr: Int): Int {
```

Comment:
We already have a util method that fetches the color from the attribute. Can we use that instead? See: https://github.com/celzero/rethink-app/blob/3d393ead467a4b4079d7b641368e618ca3b5e90f/app/src/full/java/com/celzero/bravedns/util/UIUtils.kt#L247